src/less/checkbox/base/checkbox.less

```diff
@@ -21,11 +23,13 @@ svg.checkbox__unchecked {
 }
 
 svg.checkbox__checked {
-    color: @checkbox-checked-color;
+    // color: @checkbox-checked-color;
```

Comment:
PR, looks good. Go ahead and remove the commented code.

Reply:
Got it!